onnx/defs/math/defs.cc

```diff
@@ -2815,7 +2815,7 @@ ONNX_OPERATOR_SET_SCHEMA(
           // Set the output dimension to match the dft_length on the axis.
           // If onesided this will be adjusted later on...
           const TensorProto* dft_length = nullptr;
-          if (ctx.getNumInputs() >= 2 && ctx.getInputType(1) != nullptr) {
```

Comment:
How to specify a no optional input with empty tensor so that the input is treated as no-input? e.g. scales input of Resize opset11.